src/main/java/core/basesyntax/services/DataConverter.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.services;
+

```

Comment:
Don't begin class or method implementation with empty line. Please remove this redundant empty line.